### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by nikhil-sharma-nks**
### Deployed Link - https://deploy-preview-3--silk-ui.netlify.app/components/avatar.html
image
image

### Deployed Link - https://deploy-preview-3--silk-ui.netlify.app/components/badge.html
image
image

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  font-size: 2rem;
+  width: 4rem;
+  height: 4rem;
+}
+
+.avatar-small {
+  font-size: 1.5rem;
+  width: 3rem;
+  height: 3rem;
+}
```
Comment (arjunbharti) [inline components/avatar.css:45]: Here, you have written avatar-xl. Instead, you can write avatar-extraLarge to follow the same naming convention as mentioned in the below class names.

**Review by arjunbharti — COMMENTED** [review]
UI is really good. Just added a comment, everything else is fine!

File: components/avatar.css (hunk 1 above)
Comment (nikhil-sharma-nks) [reply components/avatar.css:45]: Okay arjun👍

**Comment by nikhil-sharma-nks** [comment]
> UI is really good. Just added a comment, everything else is fine!

**Review by anandkumarcoder — APPROVED** [review]
Looks good to me

**Pull request merged**